README.md

```diff
@@ -30,6 +30,9 @@ Kubernetes has three types of groups that are officially supported:
 * **Working Groups** are temporary groups that are formed to address issues that cross SIG boundaries.
   Working groups do not own any code or other long term artifacts.
   Working groups can report back and act through involved SIGs.
+* **User Groups** are groups for facilitating communication and discovery of information related to
+  topics that have long term relevance to large groups of Kubernetes users.
+  They do not produce any deliverables or have ownership of parts of the Kubernetes code base.
```

Comment:
they could be writing white papers or blog posts etc right? how do we word this better?

Reply:
:thinking: We also say that working groups can't own code. I'm guessing if a WG writes a blog post, the "ownership" of that blog post falls under the sponsoring SIGs.

On a similar vein, maybe we could extrapolate the ownership of a blog post by a User Group to whichever SIG seems closet to the topic of the post?

Reply:
Let's just say `They do not have ownership of parts of the Kubernetes code base.` and keep it simple for now.

Reply:
Done.